src/plasma/format/plasma.fbs

```diff
@@ -39,7 +39,9 @@ enum MessageType:int {
   PlasmaUnsubscribeRequest,
   // Sending and receiving data.
```

Comment:
It'd be good to expand on the documentation for `PlasmaDataRequest` and `PlasmaDataReply`. For example, describe how this is used for transferring objects between plasma managers, and how there will be one `PlasmaDataRequest` to initiate the transfer and then multiple `PlasmaDataReply` messages containing the bytes.